### ISSUE TITLE
TypeError on GUILD_UPDATE

### ISSUE THREAD

**Issue by PalmDevs** [BirbJS/Birb#24]
**Describe the bug**

**To Reproduce**
1. Connect the bot to the API, Make sure the bot has permissions to view server settings
2. Edit any server setting
3. See the error

**Expected behavior**
No errors, guild object updates

**Errors**
```
        for (let i = 0; i < data.members.length; i++) {
                                         ^

TypeError: Cannot read property 'length' of undefined
    at Guild.build (......\Guild.js:83:42)
    at new Guild (...)
    at Object.GUILD_CREATE [as default] (......\websocket\handlers\GUILD_UPDATE.js:18:17)
    ...
    ...
```

**Platform**
- OS: Windows x64
- Node version: 16.6.0
- Birb.JS version: 1.0.0.pre.3

**Comment by PalmDevs** [comment]
There is a reason to this and it is that some properties are only sent on `GUILD_CREATE`
---
Source: https://discord.com/developers/docs/resources/guild#guild-object
![image](https://user-images.githubusercontent.com/67689562/146328124-c947cac3-4f7c-46af-99b7-f463c1e954c2.png)
![image](https://user-images.githubusercontent.com/67689562/146327725-41644858-9716-4ab6-a94a-b31cd8a0d155.png)
![image](https://user-images.githubusercontent.com/67689562/146327947-526745bf-97d9-4716-80ca-d92975227106.png)

**Comment by knokbak** [comment]
fixed as of pr #25
